### ISSUE TITLE
IIS6 Default Document Not Set Properly on Default Site

### ISSUE THREAD

**Issue by utdream** [utdream/lucee-installer#34]
A bug has been discovered with the IIS5 connection process where the default site doesn't always get updated with the default documents correctly. Further inspection of this error has to do with how IIS6 (horribly) deals with site contexts.

WORKAROUND: Add the default document manually to the Default Site. Sites you create in IIS6 AFTER the install process will still contain index.cfm in the default document list.

In short, I will not be fixing this. IIS6 is a horrible web server, and I refuse to develop using it's awful nearly useless API any longer.
